src/components/oculus-touch-controls.js

```diff
@@ -81,6 +88,13 @@ module.exports.Component = registerComponent('oculus-touch-controls', {
     this.previousButtonValues = {};
     this.bindMethods();
     this.isControllerPresent = isControllerPresent; // to allow mock
+    this.getGamepadsByPrefix = getGamepadsByPrefix; // to allow mock
+  },
+
+  update: function (oldData) {
+    if ((oldData.emulated !== undefined) && (this.data.emulated !== oldData.emulated)) {
+      this.checkIfControllersPresent();
```

Comment:
This LOC isn't getting covered by the tests. Maybe something wrong with the condition?

I think just `if (this.data.emulated !== oldData.emulated)` alone should work